### DISCUSSION TITLE
Education Verification

### DISCUSSION THREAD

**Discussion by tresaroseb** [community/community#84987] [GitHub Education]
### Hi everyone,

Hi, My current class requires the Education level of GitHub but I know I am going to run into issues with the verification and I need to find out which documentation I should use. The issue is this, I also work for the University where I am a student and since my employee status preceded my student status, my ID does not identify me as a student but as staff. I am just wondering what the best alternative documentation would be? Thank you

**Comment by hans-djalali** [comment]
Hey @tresaroseb, 
There are a lot of ways you can get verification. I am a PhD student and working at the same time for the university and had the same issue. At that time, I used my enrollment page screenshot and sent it to GitHub support because I was rejected. On the other hand, my friend used his transcript (non-official), and the application went okay. When the GitHub support emailed me, they said even a letter with the university's letterhead would be enough as well, so I guess a letter, a transcript, a time-stamped screenshot of your school portal (We have one portal as an instructor, and one portal as a student), or any other possible document that you think it proves, they will probably accept.

**Reply by tresaroseb** [reply]
Thank you!
